---
title: "Yang Lab - Code"
layout: textlay
excerpt: "Yang Lab -- Code."
sitemap: false
permalink: /code/
---

# Software

---

## [ScanITD](https://github.com/ylab-hi/ScanITD)


<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/scanitd.png" style="width: 300px"> 

ScanITD performs a stepwise seed-and-realignment procedure for internal tandem duplication (ITD) detection with accurate variant allele fraction prediction. 

- <a href="https://github.com/ylab-hi/ScanITD"><i class='fab fa-github'></i> Source</a>
- <a href="https://doi.org/10.1093/gigascience/giaa089"><i class='fa fa-book'></i> Paper</a>

---

## [ScanNeo](https://github.com/ylab-hi/ScanNeo)


<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/scanneo.png" style="width: 400px"> 

ScanNeo is a pipeline for identifying insertion and deletion (indel) introduced neoantigens from RNA sequencing data.

- <a href="https://github.com/ylab-hi/ScanNeo"><i class='fab fa-github'></i> Source</a>
- <a href="https://doi.org/10.1093/bioinformatics/btz193"><i class='fa fa-book'></i> Paper</a>

---

## [transIndel](https://github.com/cauyrd/transIndel)


<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/transindel.png" style="width: 450px"> 

transIndel is used to detect indels (insertions and deletions) from DNA-seq or RNA-seq data by parsing chimiric alignments from BWA-MEM.

- <a href="https://github.com/cauyrd/transIndel"><i class='fab fa-github'></i> Source</a>
- <a href="https://doi.org/10.1186/s12864-018-4671-4"><i class='fa fa-book'></i> Paper</a>

---

## [ScanIndel](https://github.com/cauyrd/ScanIndel)

<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/scanindel.png" style="width: 450px"> 

ScanIndel is a python program to detect indels (insertions and deletions) from NGS data by re-align and de novo assemble soft clipped reads.

- <a href="https://github.com/cauyrd/ScanIndel"><i class='fab fa-github'></i> Source</a>
- <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-015-0251-2"><i class='fa fa-book'></i> Paper</a>

---

## [EgoNet](https://github.com/cauyrd/EgoNet)

<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/egonet.png" style="width: 500px"> 

EgoNet is implemented by Python and it is designed to detecting disease related subnetwork from a large biological network (PPI, metabolic network) combined with gene expression data.

- <a href="https://github.com/cauyrd/EgoNet"><i class='fab fa-github'></i> Source</a>
- <a href="https://doi.org/10.1186/1471-2164-15-314"><i class='fa fa-book'></i> Paper</a>

---

## [ARSER](https://github.com/cauyrd/ARSER)

<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/arser.png" style="width: 500px"> 

ARSER is a Python package for identifying periodic expression profiles in analyzing circadian microarray data and has been released under the GPL.

- <a href="https://github.com/cauyrd/ARSER"><i class='fab fa-github'></i> Source</a>
- <a href="https://doi.org/10.1093/bioinformatics/btq189"><i class='fa fa-book'></i> Paper</a>
- <a href="https://www.youtube.com/watch?v=qrqwtOBlpPM&t=245s&ab_channel=ISCB"><i class="fab fa-youtube"></i> ISMB 2010 Proceeding Talk</a>

---
