<!DOCTYPE html>
<html>

  <head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Yang Lab - Code</title>
	<meta name="description" content="Yang Lab -- Code.">
	<link rel="stylesheet" href="/css/main.css">
	<link rel="canonical" href="/code/">
	<link rel="shortcut icon" type ="image/x-icon" href="/images/favicon.ico">
	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
	<link rel="stylesheet" href="/css/academicons/css/academicons.css">

	
</head>


  <body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container-fluid">
	<div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
	  </button>
	
    <a class="navbar-brand" href="/">Yang Lab @ NU</a>
	</div>

	<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li><a href="/">Home</a></li>
		<li><a href="/team">Team</a></li>
		<li><a href="/research">Research</a></li>
		<li><a href="/publications">Publications</a></li>
		<li><a href="/code">Code</a></li>
		<li><a href="/photo">Photos</a></li>
		<li><a href="/contact">Contact</a></li>
	  </ul>
	</div>
	
  </div>
</div>


    <div class="container-fluid">
      <div class="row">
        <div id="textid" class="col-sm-12">
  <h1 id="software">Software</h1>

<hr />

<h2 id="scanexitronlr"><a href="https://github.com/ylab-hi/ScanExitronLR">ScanExitronLR</a></h2>

<p><img src="/images/codepic/scanexitronlr.png" style="width: 300px" /></p>

<p>LncGSEA is a computational workflow for exitron splicing identification in long-read RNA-seq data.</p>

<ul>
  <li><a href="https://github.com/ylab-hi/ScanExitronLR"><i class="fab fa-github"></i> Source</a></li>
  <li><a href="https://doi.org/10.1093/bioinformatics/btac626"><i class="fa fa-book"></i> Paper</a></li>
</ul>

<hr />
<h2 id="lncgsea"><a href="https://github.com/ylab-hi/lncGSEA">LncGSEA</a></h2>

<p><img src="/images/codepic/lncgsea.png" style="width: 300px" /></p>

<p>LncGSEA is a convenient tool to predict the lncRNA associated pathways through Gene Set Enrichment Analysis (GSEA) of gene expression profiles from large-scale cancer patient samples.</p>

<ul>
  <li><a href="https://github.com/ylab-hi/lncGSEA"><i class="fab fa-github"></i> Source</a></li>
  <li><a href="https://doi.org/10.1186/s12864-021-07900-y"><i class="fa fa-book"></i> Paper</a></li>
</ul>

<hr />
<h2 id="scanexitron"><a href="https://github.com/ylab-hi/ScanExitron">ScanExitron</a></h2>

<p><img src="/images/codepic/scanexitron.png" style="width: 300px" /></p>

<p>ScanExitron is a computational workflow for exitron splicing identification from RNA-seq.</p>

<ul>
  <li><a href="https://github.com/ylab-hi/ScanExitron"><i class="fab fa-github"></i> Source</a></li>
  <li><a href="https://doi.org/10.1016/j.molcel.2021.03.028"><i class="fa fa-book"></i> Paper</a></li>
  <li><a href="https://star-protocols.cell.com/protocols/977"><i class="fa fa-book"></i> Protocol</a></li>
  <li><a href="https://data.mendeley.com/datasets/vdkpfzjjvg/1"><i class="fas fa-database"></i> Detected exitrons in TCGA</a></li>
</ul>

<hr />

<h2 id="scanitd"><a href="https://github.com/ylab-hi/ScanITD">ScanITD</a></h2>

<p><img src="/images/codepic/scanitd.png" style="width: 300px" /></p>

<p>ScanITD performs a stepwise seed-and-realignment procedure for internal tandem duplication (ITD) detection with accurate variant allele fraction prediction.</p>

<ul>
  <li><a href="https://github.com/ylab-hi/ScanITD"><i class="fab fa-github"></i> Source</a></li>
  <li><a href="https://doi.org/10.1093/gigascience/giaa089"><i class="fa fa-book"></i> Paper</a></li>
</ul>

<hr />

<h2 id="scanneo"><a href="https://github.com/ylab-hi/ScanNeo">ScanNeo</a></h2>

<p><img src="/images/codepic/scanneo.png" style="width: 400px" /></p>

<p>ScanNeo is a pipeline for identifying insertion and deletion (indel) introduced neoantigens from RNA sequencing data.</p>

<ul>
  <li><a href="https://github.com/ylab-hi/ScanNeo"><i class="fab fa-github"></i> Source</a></li>
  <li><a href="https://doi.org/10.1093/bioinformatics/btz193"><i class="fa fa-book"></i> Paper</a></li>
</ul>

<hr />

<h2 id="transindel"><a href="https://github.com/cauyrd/transIndel">transIndel</a></h2>

<p><img src="/images/codepic/transindel.png" style="width: 450px" /></p>

<p>transIndel is used to detect indels (insertions and deletions) from DNA-seq or RNA-seq data by parsing chimiric alignments from BWA-MEM.</p>

<ul>
  <li><a href="https://github.com/cauyrd/transIndel"><i class="fab fa-github"></i> Source</a></li>
  <li><a href="https://doi.org/10.1186/s12864-018-4671-4"><i class="fa fa-book"></i> Paper</a></li>
</ul>

<hr />

<h2 id="scanindel"><a href="https://github.com/cauyrd/ScanIndel">ScanIndel</a></h2>

<p><img src="/images/codepic/scanindel.png" style="width: 450px" /></p>

<p>ScanIndel is a python program to detect indels (insertions and deletions) from NGS data by re-align and de novo assemble soft clipped reads.</p>

<ul>
  <li><a href="https://github.com/cauyrd/ScanIndel"><i class="fab fa-github"></i> Source</a></li>
  <li><a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-015-0251-2"><i class="fa fa-book"></i> Paper</a></li>
</ul>

<hr />

<h2 id="egonet"><a href="https://github.com/cauyrd/EgoNet">EgoNet</a></h2>

<p><img src="/images/codepic/egonet.png" style="width: 500px" /></p>

<p>EgoNet is implemented by Python and it is designed to detecting disease related subnetwork from a large biological network (PPI, metabolic network) combined with gene expression data.</p>

<ul>
  <li><a href="https://github.com/cauyrd/EgoNet"><i class="fab fa-github"></i> Source</a></li>
  <li><a href="https://doi.org/10.1186/1471-2164-15-314"><i class="fa fa-book"></i> Paper</a></li>
</ul>

<hr />

<h2 id="arser"><a href="https://github.com/cauyrd/ARSER">ARSER</a></h2>

<p><img src="/images/codepic/arser.png" style="width: 500px" /></p>

<p>ARSER is a Python package for identifying periodic expression profiles in analyzing circadian microarray data and has been released under the GPL.</p>

<ul>
  <li><a href="https://github.com/cauyrd/ARSER"><i class="fab fa-github"></i> Source</a></li>
  <li><a href="https://doi.org/10.1093/bioinformatics/btq189"><i class="fa fa-book"></i> Paper</a></li>
  <li><a href="https://www.youtube.com/watch?v=qrqwtOBlpPM&amp;t=245s&amp;ab_channel=ISCB"><i class="fab fa-youtube"></i> ISMB 2010 Proceeding Talk</a></li>
</ul>

<hr />

</div>

      </div>
    </div>

    <div id="footer" class="panel">
  <div class="panel-footer">
	<div class="container-fluid">
	  <div class="row">


			<ul class="nav navbar-nav navbar-right">
				<li><a href="https://www.feinberg.northwestern.edu/">Feinberg School of Medicine</a></li>
				<li><a href="https://yanglab-guide.readthedocs.io/en/latest/">Lab Guide</a></li>
				<li><a href="/about">About</a></li>
				<li><a href="/recruitment">Recruitment</a></li>
			</ul>

		
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="/js/bootstrap.min.js"></script>


  </body>

</html>
